### PR TITLE
Issue #11003: upgrade http to https for xsd links

### PR DESCRIPTION
**Pull request by adityavishwakkarma**
#11003
changed links for xsd files from http to https

### CODE REVIEW

File: config/deploy-settings.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
           xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0
-                              http://maven.apache.org/xsd/settings-1.0.0.xsd">
+                              https://maven.apache.org/xsd/settings-1.0.0.xsd">
```
Comment (romani) [inline config/deploy-settings.xml:4]: Can we update xsi:schemaLocation also?

**Review by romani — CHANGES_REQUESTED** [review]
Item:

File: config/deploy-settings.xml (hunk 1 above)
Comment (adityavishwakkarma) [reply config/deploy-settings.xml:4]: What should be the value of xsi:schemaLocation

Comment (romani) [reply config/deploy-settings.xml:4]: Https

Comment (adityavishwakkarma) [reply config/deploy-settings.xml:4]: @romani xsi:schemaLocation is to be changed for all files or for file only

Comment (romani) [reply config/deploy-settings.xml:4]: For all files

**Comment by romani** [comment]
Keep in your  single commit.
Read https://checkstyle.org/beginning_development.html#Starting_Development

**Comment by romani** [comment]
Squash all commits in one

File: config/deploy-settings.xml (hunk 1 above)
Comment (adityavishwakkarma) [reply config/deploy-settings.xml:4]: @romani 
Error "cvc-elt.1.a: Cannot find the declaration of element 'project'." 
is shown on vs code while changing value of xsi:schemaLocation

Comment (romani) [reply config/deploy-settings.xml:4]: Ok, sad .

Please fix commits, by squashing all in one to let me merge this Place

**Comment by romani** [comment]
Github, rebase

**Comment by romani** [comment]
Github, rebase

**Comment by adityavishwakkarma** [comment]
@romani I made a new pull request #11057 
please check it

**Pull request closed without merging**